tsconfig.json

```diff
@@ -19,13 +16,9 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
-    "target": "esnext"
+    "target": "esnext",
+    "incremental": true
```

Comment:
why did you add `incremental: true`?

Reply:
When I upgraded to v12, and then did `yarn dev`, the log statements said that it configured the tsconfig.json file. The only thing added was `"incremental": true`

Reply:
ah it stores info about the complied files: https://www.typescriptlang.org/tsconfig#incremental